src/pages/student/categories/CategoryList/index.js

```diff
@@ -37,28 +37,31 @@ function CategoryList() {
     return categories.map((category, idx) => {
       return (
         <Card className={style.card} key={idx}>
-          <Card.Header id={style.cardHeader}></Card.Header>
+          <Card.Header id={style.cardHeader}>
+            <p className={style.cardTitle}>{category.name}</p>
+            <p className={style.cardSubtitle}>{category.description}</p>
+          </Card.Header>
           <Card.Body>
             {category.subcategories_count ? (
               <Link to={`/categories/${category.id}/sub`}>
-                <div className={style.cardContent}>
-                  <center>
-                    {' '}
-                    <Card.Title className={style.cardTitle}>
-                      {category.name}
-                    </Card.Title>{' '}
-                  </center>
+                <div>
+                  {' '}
+                  <Card.Title >
+                    <a id={style.Subtitle}>
+                        View Available Subcategories: 5
+                    </a>
+                  </Card.Title>{' '}
                 </div>
               </Link>
             ) : (
               <Link to={`/categories/${category.id}/quizzes`}>
-                <div className={style.cardContent}>
-                  <center>
-                    {' '}
-                    <Card.Title className={style.cardTitle}>
-                      {category.name}
-                    </Card.Title>{' '}
-                  </center>
+                <div>
+                  {' '}
+                  <Card.Title>
+                    <a id={style.Subtitle}>
+                        View Available Subcategories: 6
```

Comment:
- [ ] Change the link label to : "Check Available Quizzes" since this link will redirect to quiz list page.